### ISSUE TITLE
Relabel navigation and help buttons/icons to improve accessibility

### ISSUE THREAD

**Issue by JeffH-AWS** [opensearch-project/OpenSearch-Dashboards#3557]
**Is your feature request related to a problem? Please describe.**

The OpenSearch Dashboards web interface uses buttons that are ambiguous to users, which makes it unclear how to document processes that include interactions with these UI elements:

![image](https://user-images.githubusercontent.com/107883031/223779599-3a468c04-5e92-4683-8c25-9ffd73490aba.png)

**Describe the solution you'd like**

In order to improve usability, accessibility, and in order to facilitate product documentation, these icon-buttons should be either labeled or replaced with icons that include text that can be referenced.

For example, according to a member of the UX team the weird icon in the top-right corner is called the "Help" icon. How on earth is anyone going to interpret that aside from relying on arbitrary past experiences that might make us look at the top right corner? This type of behavior shouldn't be so ambiguous.

Some websites even argue that the menu icon should be in the top-right. See https://uxpickle.com/hamburger-menu-icon-should-it-be-on-the-left-or-right/

**Describe alternatives you've considered**

Establishing a dictionary of icons used in the OSD UI so that there's a secret decoder ring built into the product.

**Additional context**

The lack of defined terms for these UI elements is resulting in inconsistent documentation.

**Comment by joshuarrrr** [comment]
@kgcreative @KrooshalUX to review

**Comment by KrooshalUX** [comment]
@JeffH-AWS  Thanks for the issue - a few things of note:

- The menu indicator (three horizontal lines in the upper left corner) is fairly prevalent across web and mobile applications, however I noticed that the appropriate browser tooltip is missing and I will create an issue for that. 
- The UX team is currently redesigning many of the icons within OUI in order to bring clarity to items such as "help" which is currently visually explained using a "life bouy" which is an outdated UI practice. We are specifically tracking that issue here: https://github.com/opensearch-project/ux/issues/11 . Noted that the tooltip for Help is also missing and will add it to the issue for header tooltips. 
- You can refer to the icon names here: https://oui.opensearch.org/1.0/#/display/icons
- The Design System team is working on documentation for how/when to use iconography and that will be provided via a documentation website in the near future. Until then, please feel free to reach out for clarification to myself or any UX partner. 
- Lastly, I am not sure where you are seeing what used to be the "home" button, which has now been updated to the OpenSearch logo to match the practice of clicking logo = directs a user to "home". That one is properly labelled with a tooltip.
<img width="1791" alt="Screen Shot 2023-03-09 at 10 02 51 AM" src="https://user-images.githubusercontent.com/105819041/224117207-5535061f-e3b4-4983-98ba-a25c977584f0.png">

**Comment by JeffH-AWS** [comment]
@KrooshalUX wow that is awesome! Thanks for the detailed context and references!! I'll make sure to follow progress and leverage that awesome icon name reference while I write.

**Comment by KrooshalUX** [comment]
@JeffH-AWS  happy to help. Closing this in favor of https://github.com/opensearch-project/OpenSearch-Dashboards/issues/3573 and our continued icon redesign efforts.
